spec/controllers/qualifications_controller.rb

```diff
@@ -26,4 +25,32 @@
       end
     end
   end
+
+  context 'POST create' do
+    let(:qualification_params) { { qualification: { start_at: '01.03.2024', finish_at: '31.03.2024' } } }
+
+    it 'does not permit finish_at for qualification kinds with validity' do
```

Comment:
Besser als `permit` fände ich `ignores` und `allows` for qualifications with usw .. entspricht imho mehr dem was eigentlich passiert